tests/phpunit/includes/queryprinters/ListResultPrinterTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+namespace SMW\Test;
+
+use SMW\ListResultPrinter;
+
+/**
+ * @covers \SMW\ListResultPrinter
+ *
+ * @group SMW
+ * @group SMWExtension
+ *
+ * @license GNU GPL v2+
+ * @since 3.0
+ */
+class ListResultPrinterTest extends QueryPrinterTestCase {
+	/**
+	 * @dataProvider allFormatsProvider
+	 * @param string $format
+	 */
+	public function testCanConstruct( $format ) {
+		$listResultPrinter = new ListResultPrinter( $format );
+
+		$this->assertInstanceOf( '\SMW\ListResultPrinter', $listResultPrinter );
+	}
+
+	/**
+	 * @dataProvider listFormatProvider
```

Comment:
I recommend putting the data providers right below the method that uses them, so less vertical scrolling is needed.